staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -1665,12 +1666,32 @@ func getPodsForPVC(c corev1client.PodInterface, pvcName string, settings Describ
 
 	for _, pod := range nsPods.Items {
 		for _, volume := range pod.Spec.Volumes {
-			if volume.VolumeSource.PersistentVolumeClaim != nil && volume.VolumeSource.PersistentVolumeClaim.ClaimName == pvcName {
+			if volume.VolumeSource.PersistentVolumeClaim != nil && volume.VolumeSource.PersistentVolumeClaim.ClaimName == pvc.Name {
 				pods = append(pods, pod)
 			}
 		}
 	}
 
+	for _, ownerRef := range pvc.ObjectMeta.OwnerReferences {
+		if ownerRef.Kind != "Pod" {
+			continue
+		}
+
+		podIndex := slices.IndexFunc(nsPods.Items, func(pod corev1.Pod) bool {
```

Comment:
Can we just iterate with a for loop rather than bringing in external library doing just that?

Reply:
You're right. Just applied it.